### ISSUE TITLE
Darksiders II Deathinitive Edition - graphical glitches

### ISSUE THREAD

**Issue by ziabice** [doitsujin/dxvk#606]
I'm trying this game using Steam Play. The game works well (needs the `PROTON_NO_ESYNC` option to work), but has some graphical glitches here and there.

Here's a screenshot of the glitches, taken at the end of the first level:
![screenshot_20180826_204740](https://user-images.githubusercontent.com/28961115/44632048-a7201c00-a974-11e8-9575-3cc957afe130.png)

### Software information
Darksiders II - Deathinitive Edition, with Steam Play and Proton 3.7-3

### System information
- GPU: AMD RX 580 4GB
- Driver: Mesa 18.1.6
- Wine version: Proton 3.7-3
- DXVK version: should be 0.70

### Log files
Were are these files on Proton?

- d3d11.log:
- dxgi.log:

**Comment by varris1** [comment]
The logs should be where the .exe resides. For issues like these, you should record a trace on Windows if you can.

**Comment by doitsujin** [comment]
Please record an apitrace, can't do anything without being able to reproduce the bug locally.

**Comment by ziabice** [comment]
I've read the dxvk wiki and found the DLL to do the apitrace, I'll try ASAP!

**Comment by doitsujin** [comment]
Is this even a D3D11 game? All info I can find indicates that it is in fact D3D9 **only**.

**Comment by mati865** [comment]
Darksiders II was DX9 only but remastered Deathinitive Edition should have DX11: https://steamdb.info/app/388410/config/

**Comment by tingvarsson** [comment]
Are you starting it with the -dx11 flag that is mentioned at steamdb? Could be worth a try.

**Comment by ziabice** [comment]
Sorry for the delay in my reply, returning from work I did some testing. 
First of all, the game has different renders: DX9, DX11, OpenGL. The DX9 is the default renderer. 

To enable the OpenGL renderer I set these launch options: `PROTON_NO_ESYNC=1 %command% -gl`

To enable the DirectX11renderer I set these launch options: `PROTON_NO_ESYNC=1 %command% -dx11`

OpenGL renderer works well, has the same glitches that the DX9. The DirectX11 renderer fails to start: a window appears with the message "Could not read your game shader library located ...".

Playing with video settings I discovered the source of the problem: antialiasing. If set to OFF or to MAX, there are no visual glitches, other than that causes the problems you can see into my image.

So, apart from the DX11 renderer not working at all, this is definitely not a DXVK issue.

I haven't time to test the game on a vanilla WINE.

**Comment by doitsujin** [comment]
> So, apart from the DX11 renderer not working at all, this is definitely not a DXVK issue.

Closing then. If the Dx11 renderer is not officially supported for the game, it's better to use the default Dx9 path anyway because there might be similar issues as with the OpenGL path.

**Comment by FOSSHipster** [comment]
I have tried pretty much everything to get this game to work. It keeps crashing after the videos for the companies involved are done playing. I have tried different Wine versions, and even switched distros to see if that is the problem. I am currently trying to get it to run on Manjaro with an NVIDIA graphics card. Maybe because this is the Epic Games Store version through Heroic that I am having issues.
